src/hashTable.js

```diff
@@ -0,0 +1,125 @@
+class Node {
+  constructor(key, value) {
+    this.key = key,
+    this.value = value,
+    this.next = null
+  }
+}
+
+export default class HashTable {
+  constructor() {
+    this._length = 0
+    this.valueStore = []
```

Comment:
if you are encapsulating ```_length``` here on line 11 you would probably want to do that on line 12 to ```valueStore``` too. I'm not too familiar with this naming convention...had to ask @jrob about it.

Would love to chat about you reasoning if there is time. Or once you get the tests to pass.